research-app/src/App.vue

```diff
@@ -1978,4 +2139,19 @@ ul.tool-menu {
   }
 }
 
+.pointer {
+  cursor: pointer;
+}
+
+/**
+This makes the last element of the last list item in the
+display panel have the rounded bottom edge
+The alternative to this is to have Vue bind a class to the last element
+Since I'm using
```

Comment:
Sentence fragment in the comment here.

Reply:
I honestly have no clue what I was going to write, so I just removed that line.